### PR TITLE
Activity: Set a deadline for subscription for an activity

### LINKED ISSUE #192: Activity: Set a deadline for subscription for an activity
Allow the subscriptions to stop before the actual activity.

### PR DESCRIPTION
**Pull request by stefanmajoor**
Fixes #192

### CODE REVIEW

File: module/Activity/src/Activity/Controller/ActivityController.php [hunk 1]
```diff
@@ -176,13 +174,4 @@ public function signoffAction()
...
     }
-
-    public function touchAction()
-    {
-        $viewModel = new ViewModel();
-        $viewModel->setVariables(array('key' => 'value'))
-            ->setTerminal(true);
-
-        return $viewModel;
-    }
```
Comment (jszanto) [inline module/Activity/src/Activity/Controller/ActivityController.php]: Why was this removed?

**Comment by jszanto** [comment]
It appears something went wrong when you merged the master in to this branch (which is a bad practice too).

**Comment by stefanmajoor** [comment]
Something went horribly wrong when the changes from the master were applied. Let me have some time to figure it out, and I will make another pull request if I have fixed what went wrong.

Also git automatically merges if you pull. That is why a merge was doen.

**Comment by jszanto** [comment]
The easiest would be to remove that last merge commit, fetch the master (instead of pulling) and then rebase upon the master,

**Comment by stefanmajoor** [comment]
Except it did not go wrong in the merge, but rather stuff just disappeared in the first commit I made at that branch.
